tests/python/tests.py

```diff
@@ -149,8 +153,45 @@ def test1():
     print election_result(userA, election)
     print election_result(userB, election)
 
-def main():
+def test2():
```

Comment:
@neckberg this integration test attempts to duplicate the problems you described in #54. Unfortunately I was not able to reproduce the problem. Would you be able to customize this test to reproduce it?